File: upload-to-drive.js
const { chromium } = require('playwright');
const path = require('path');

(async () => {
  console.log('🚀 Starting Google Drive upload automation...');
  
  const browser = await chromium.launch({ 
    headless: false,
    slowMo: 100
  });
  
  const context = await browser.newContext({
    viewport: { width: 1280, height: 800 }
  });
  
  const page = await context.newPage();
  
  // Go to Google Drive
  console.log('📂 Opening Google Drive...');
  await page.goto('https://drive.google.com');
  
  // Wait for user to be logged in
  console.log('⏳ Waiting for login...');
  await page.waitForSelector('[role="button"], [aria-label*="New"]', { timeout: 60000 });
  
  console.log('✅ Google Drive loaded!');
  console.log('');
  console.log('📋 Manual steps to complete:');
  console.log('1. Make sure you are logged in as Borat14011@gmail.com');
  console.log('2. Click "New" → "Folder"');
  console.log('3. Name it: Trading-Strategy-Presentation');
  console.log('4. Open the folder');
  console.log('5. Click "New" → "File upload"');
  console.log('6. Select these files:');
  console.log('   - C:\\Users\\Borat\\.openclaw\\workspace\\netlify-deploy\\index.html');
  console.log('   - C:\\Users\\Borat\\.openclaw\\workspace\\PROFESSIONAL_STRATEGY_PRESENTATION.md');
  console.log('7. Right-click folder → "Share" → "Copy link"');
  console.log('');
  console.log('🎯 Files ready to upload:');
  console.log('- index.html (web presentation)');
  console.log('- PROFESSIONAL_STRATEGY_PRESENTATION.md (markdown source)');
  
  // Keep browser open
  console.log('');
  console.log('✨ Browser will stay open. Press Ctrl+C to close.');
  
})();

File: upload-to-drive.py
from playwright.sync_api import sync_playwright
import time

def upload_to_google_drive():
    print("Starting Google Drive upload automation...")
    
    with sync_playwright() as p:
        # Launch browser
        browser = p.chromium.launch(headless=False, slow_mo=100)
        context = browser.new_context(viewport={'width': 1280, 'height': 800})
        page = context.new_page()
        
        # Go to Google Drive
        print("Opening Google Drive...")
        page.goto('https://drive.google.com')
        
        # Wait for page to load
        print("Waiting for Google Drive to load...")
        page.wait_for_selector('text=New', timeout=60000)
        
        print("Google Drive loaded!")
        print("")
        print("Next steps:")
        print("1. Click 'New' -> 'Folder'")
        print("2. Name it: Trading-Strategy-Presentation")
        print("3. Open the folder")
        print("4. Click 'New' -> 'File upload'")
        print("5. Navigate to: C:\\Users\\Borat\\.openclaw\\workspace\\netlify-deploy\\")
        print("6. Select: index.html")
        print("7. Repeat for: PROFESSIONAL_STRATEGY_PRESENTATION.md")
        print("8. Right-click folder -> 'Share' -> 'Copy link'")
        print("")
        print("Files ready:")
        print("- index.html (web presentation)")
        print("- PROFESSIONAL_STRATEGY_PRESENTATION.md (markdown)")
        print("")
        print("Browser will stay open. Close manually when done.")
        
        # Keep browser open
        while True:
            time.sleep(1)

if __name__ == "__main__":
    upload_to_google_drive()
